### PR TITLE
Updated file for fixing IOTAUL_issue391

### PR DESCRIPTION
**Pull request by Madhu-NEC**
Fixes the issue [IOTA-UL-391](https://github.com/telefonicaid/iotagent-ul/issues/391) for listing of service groups.

### CODE REVIEW

File: lib/services/groups/groupRegistryMemory.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ var registeredGroups = {},
     logger = require('logops'),
```
Comment (fgalan) [inline lib/services/groups/groupRegistryMemory.js:26]: Please add an entry in CHANGES_NEXT_RELEASE corresponding to the fix done in this PR.

**Comment by AlvaroVega** [comment]
This PR is for groupRegistryMemory.js, but what about when mongodb is configured for registry ?

**Comment by Madhu-NEC** [comment]
> This PR is for groupRegistryMemory.js, but what about when mongodb is configured for registry ?

Hi @AlvaroVega ,
As per the [issue comment](https://github.com/telefonicaid/iotagent-ul/issues/391#issuecomment-546673623) the issue is reproduced only when the registry is configured as memory. Issue does not exist for mongodb. I have also confirmed the above case by listing the service groups while configuring the registry as mongodb.

**Review by AlvaroVega — APPROVED** [review]
LGTM

**Review by fgalan — APPROVED** [review]
LGTM

Please solve the conflict in CHANGES_NEXT_RELEASE file so this PR can be merged.

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -1,3 +1,4 @@
+Fix: Listing service groups returns only a single service group (iota-ul #391)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
Fix: Listing service groups returns only a single service group (#894)
```

Af far as I understand, this issue is in the iotagent-node-lib librery, affecting not only IOTA-UL but other agents. Thus I have transferred it to library repo and issue number is now other.

**Pull request merged**